internal/namespaces/baremetal/v1alpha1/custom_server.go

```diff
@@ -0,0 +1,99 @@
+package baremetal
+
+import (
+	"context"
+	"reflect"
+	"time"
+
+	"github.com/scaleway/scaleway-cli/internal/core"
+	baremetal "github.com/scaleway/scaleway-sdk-go/api/baremetal/v1alpha1"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+const (
+	serverActionTimeout = 20 * time.Minute
+)
+
+type baremetalActionRequest struct {
+	ServerID string
+	Zone     scw.Zone
+}
+
+var serverActionArgSpecs = core.ArgSpecs{
```

Comment:
Are they use for something else than wait? If not move them in the command directly